src/routes/anime/[id]/+page.svelte

```
@@ -0,0 +1,640 @@
+                                        class="form-input"
+                                    />
+                                </label>
+                            {:else}
+                                <input type="hidden" name="progress" value={progress} />
+                            {/if}
+                        </div>
+
+                        <div class="form-actions">
+                            <button type="submit" class="btn-primary {data.anime.user_entry ? 'btn-primary--update' : 'btn-primary--add'}">
+                                {#if data.anime.user_entry}
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"><path d="M20 6L9 17l-5-5"/></svg>
```

Biome (2.4.13)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)